payjoin-cli/src/main.rs

```diff
@@ -111,6 +111,12 @@ fn cli() -> ArgMatches {
     let mut cmd = cmd.subcommand(Command::new("resume"));
 
     // Conditional arguments based on features for the receive subcommand
+    receive_cmd = receive_cmd.arg(
+        Arg::new("max_feerate")
+            .long("max-feerate")
```

Comment:
I think max_fee_rate is more descriptive since it's not necessarily the actual tx feerate nor is it always the actual feerate paid by the receiver for their contributions (e.g if the sender partially covered fees on the input side). Fixed the parameter name for dash/underscore consistency.